burn-tch/src/ops/base.rs

```diff
@@ -439,4 +439,16 @@ impl<E: tch::kind::Element + Copy + Default> TchOps<E> {
             .map(|tensor| TchTensor::new(tensor))
             .collect()
     }
+    pub fn pow<const D: usize>(
+        tensor: TchTensor<E, D>,
+        exponent: TchTensor<E, D>,
+    ) -> TchTensor<E, D> {
+        TchTensor::binary_ops_tensor(
+            tensor,
+            exponent,
+            |lhs, rhs| lhs.f_pow_tensor_(rhs).unwrap(),
+            |lhs, rhs| rhs.f_pow_tensor_(lhs).unwrap(),
```

Comment:
I don't think it's equivalent, we should use the readonly version in this case.

Reply:
I think I resolved it, but just to clarify, you mean switching to `f_pow` (which takes `&self` rather than `&mut self`) right?